### PR TITLE
Re-enable auto-layout tests

### PR DESCRIPTION
**Pull request by teunhoevenaars**
### PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I have read, and I am following the [Contributor guide](https://github.com/gaphor/gaphor/blob/main/CONTRIBUTING.md)
- [x] I have read, and I understand the GNOME [Code of Conduct](https://conduct.gnome.org/)

### PR Type
What kind of change does this PR introduce?

- [x] Bug fix
- [ ] Feature
- [ ] Chore (refactoring, formatting, local variables, other cleanup)
- [ ] Documentation content changes

### What is the current behavior?
test is disabled due to large amount of time required

Issue Number: #2104 

### What is the new behavior?
test completes in few seconds. Realized by setting the hypothesis testing to limit itself to generating examples (and not compare, reduce etc), see https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.Phase. Also limit the number and extend of tests.   

### Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### Other information
Feel free to adjust the setting parameters; I just picked something that achieved completion in few seconds and which I believed made sense.

### CODE REVIEW

File: tests/test_auto_layouting.py [hunk 1]
```diff
@@ -97,7 +92,8 @@ def select(self, expression=None):
         assume(elements)
         return sampled_from(elements)
 
-    def relations(self, diagram):
+    @staticmethod
```
Comment (danyeaw) [inline tests/test_auto_layouting.py:95]: I know IDEs often provide guidance that a method could be static. I would avoid this unless you want to directly call AutoLayouting.relations(). You could also move it from a method to a function outside of this class.

**Review by danyeaw — COMMENTED** [review]
Hi @teunhoevenaars, thanks so much for the contribution to help us get our test turned back on. I just had one more comment.

@amolenaar, you are much more the hypothesis expert, any other suggestions?

File: tests/test_auto_layouting.py (hunk 1 above)
Comment (teunhoevenaars) [reply tests/test_auto_layouting.py]: thanks @danyeaw, I indeed followed my IDE's suggestion on this without too much thought. I agree with your suggestion to move it outside the class, which is now implemented.

**Review by danyeaw — APPROVED** [review]
Thanks @teunhoevenaars!!

**Pull request merged**